lib/presentation/pages/home/partial/current_match_widget.dart

```diff
@@ -0,0 +1,185 @@
+                                        Text(
+                                          match.homeClub.name!,
+                                          textAlign: TextAlign.center,
+                                          style: const TextStyle(
+                                            fontSize: 14,
+                                          ),
+                                          // softWrap: true,
+                                          overflow: TextOverflow.ellipsis,
+                                          maxLines: 1,
+                                        ),
+                                      ],
+                                    ),
+                                  ),
+                                  Text(
+                                    ' ${match.homeScore}  -  ${match.awayScore} ',
+                                    style: const TextStyle(
+                                      fontSize: 24,
+                                    ),
+                                  ),
+                                  SizedBox(
+                                    width: 100,
+                                    height: 125,
+                                    child: Column(
+                                      mainAxisAlignment:
+                                          MainAxisAlignment.center,
+                                      children: [
+                                        ClubLogoWidget(
+                                          imageUrl: match.awayClub.logo!,
+                                          width: 80,
+                                          height: 80,
+                                        ),
+                                        const SizedBox(height: 6),
+                                        Text(
+                                          match.awayClub.name!,
+                                          textAlign: TextAlign.center,
+                                          style: const TextStyle(
+                                            fontSize: 14,
+                                          ),
+                                          // softWrap: true,
+                                          overflow: TextOverflow.ellipsis,
+                                          maxLines: 1,
+                                        ),
+                                      ],
+                                    ),
+                                  ),
+                                ],
+                              );
+                            },
+                          ),
+                        ),
+                      ),
+                    ),
+                  ),
+                ),
+                Positioned(
+                  top: 18,
+                  left: 0,
+                  right: 0,
+                  child: Center(
+                    child: Text(
+                      match.location,
+                    ),
```

Comment:
yg pendek yaka ini jgn kasih coma biar jadi 1 line
```suggestion
                    child: Text(match.location),
```